### ISSUE TITLE
Meepo (and probably other heroes) Auto Casting abilities are being toggled unintentionally

### ISSUE THREAD

**Issue by eddieatjollyroger** [ValveSoftware/Dota2-Gameplay#12919]
### Ability name

Poof

### Description

If you tap CTRL + TAB (or your keybind to cycle through control groups), it toggles the auto casting ability... 
Had a really hard time understanding why Auto Casting was being turned On and killing me...
At least give us a way to disable Poof Auto Casting completely 

### Example Match ID (and possibly Timestamp)

Any meepo match right now

**Comment by eddieatjollyroger** [comment]
Accidently closed it, its still very much ocurring and it is quite urgent!

**Comment by voddov** [comment]
I have my next unit hotkey on mouse 4 and don't have your issue here, but i do have my Alt+1 control group glitched that it always selects some non-existing meepos every game.

**Comment by eddieatjollyroger** [comment]
@voddov 
Thats weird, I can replicate with Mouse4, the timing is quite specific it needs to be a very quick tap of both CTRL and Mouse4 at the same time.
Though maybe its MacOS only? I unfortunately don't have access to a windows machine to test. I posted it on the MacOS bug tracker as well anyways.
Thanks for your help!

**Comment by voddov** [comment]
I'm on windows, that might be the reason i can't replicate it? Anyways, no problem brother.

**Comment by eddieatjollyroger** [comment]
Yeah seems like there are multiple ways to trigger it, something is very wrong :sweat_smile:

**Comment by Fausto1486** [comment]
This bug is still very much present.
Also, this is an ALT cast and it should be on ALT not on CTRL which is a special key used for many different options.
The overall implementation of this UI component is, clearly, bugged and a rollback is clearly necessary.
If rolling back is not an option, at least the chance to enable/disable this functionality in the settings should be made available by default.

**Comment by eddieatjollyroger** [comment]
It probably is still present idk i dont play dota anymore im done re learning the shit UX on this hero everytime they remember it exists.
the stupidest part was that it didnt have a toggle like wtf just completely braindead

**Comment by eddieatjollyroger** [comment]
stale like the game lol
